server/tests/audit_math/test_supersimple.py

```diff
@@ -643,7 +643,7 @@ def test_multiplicity():
     "Contest B": 60000,
     "Contest C": 36000,
     "Contest D": 15000,
-    "Contest E": 6,
+    "Contest E": 33,
```

Comment:
Wow, I kinda can't believe it only affected this one contest.

Reply:
I guess that's more a testament to my testing strategy than anything, huh :trollface: